### DISCUSSION TITLE
Unable to edit img url input of mv-popup, when using Shoelace webcomponents

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#901] [Q&A]
Hi

I've made a [codepen](https://codepen.io/galinhalx/pen/eYMLvBe) , 
where after pressing the button to open drawer
I'm able to edit properties, and input like checkbox

But, I don't understand why, I'm not able to edit `img` source with Mavo: no popup to edit the source attribute

I even tryed to force it with `mv-edit-type="popup"`, but it doesn't work

Am I missing something? Or does mavo needs some extra information?

**Comment by DmitrySharabin** [comment]
Try adding the following rule to your stylesheet:

```css
.mv-popup {
	z-index: 1000;
}
```

It looks like the drawer has a rather big value for `z-index`, and we need to _beat_ it.

**Reply by GalinhaLX** [reply]
You save my day! I almost get crazy about this one....
thanks a lot @DmitrySharabin

**Reply by DmitrySharabin** [reply]
My pleasure! :)

**Comment by GalinhaLX** [comment]
Hi

Well, I was to fast to shout victory

In fact, 
1/ pop-up appears
2/ I'm able to click on upload button to upload a file
3/ I'm able to drag & drop a file , and upload it

BUT I'm not able to click & edit the 'url input' -> the popup vanishes :(

I tried to do my best with `z-index`, even with something like
```
input.mv-output {
z-index:200000!important;}
```
[codepen here](https://codepen.io/galinhalx/pen/NWYoOgq)
